static/app/components/loadingTriangle.tsx

```diff
@@ -37,8 +41,19 @@ const CircleBackground = styled('div')`
   display: flex;
   align-items: center;
   justify-content: center;
-  background: ${p => p.theme.surface300};
+  background: #fff;
```

Comment:
matches the 1st loading state

Reply:
Nit, but I would add a comment about this as it's not obvious why we set #fff here

Reply:
i was wrong about using surface100 or whatever in the other pr it's actually just white https://github.com/getsentry/sentry/blob/master/static/less/shared-components.less#L512